pkgs/development/web/cypress/default.nix

```diff
@@ -67,11 +68,16 @@ in stdenv.mkDerivation rec {
     printf '{"version":"%b"}' $version > $out/bin/resources/app/package.json
     # Cypress now looks for binary_state.json in bin
     echo '{"verified": true}' > $out/binary_state.json
-    ln -s $out/opt/cypress/Cypress $out/bin/Cypress
-
+    ln -s $out/opt/cypress/Cypress $out/bin/cypress
     runHook postInstall
   '';
 
+  postFixup = ''
+    # exit with 1 after 25.05
```

Comment:
IMO we should just drop it at that point and people will find the command not found

Reply:
Having a hard time getting it to build with the wrapper so this would make me happy...